### PR TITLE
disable tests unstable on Windows 11

### PR DESCRIPTION
**Pull request by wfurt**
contributes to #58927

cc: @ViktorHofer

### CODE REVIEW

**Review by ViktorHofer — APPROVED** [review]

File: src/libraries/System.Net.Security/tests/FunctionalTests/SslStreamStreamToStreamTest.cs [hunk 1]
```diff
@@ -66,11 +67,18 @@ public static IEnumerable<object[]> SslStream_StreamToStream_Authentication_Succ
             }
         }
 
-        [Theory]
+        [ConditionalTheory]
```
Comment (antonfirsov) [inline src/libraries/System.Net.Security/tests/FunctionalTests/SslStreamStreamToStreamTest.cs:70]: Why `ConditionalTheory` if there are no actual conditions passed to it?

**Comment by karelz** [comment]
Unrelated test failure in [System.ComponentModel.EventBasedAsync.Tests](https://helixre8s23ayyeko0k025g8.blob.core.windows.net/dotnet-runtime-refs-pull-59135-merge-e04af903171341aab4/System.ComponentModel.EventBasedAsync.Tests/1/console.10be78a7.log?sv=2019-07-07&se=2021-10-05T03%3A52%3A32Z&sr=c&sp=rl&sig=K31l1ATeY4KIMn8LWeAWAL7n40NYS9C7TB8trR4pW0I%3D):
```
    System.ComponentModel.EventBasedAsync.Tests.BackgroundWorkerTests.TestCancelAsync [FAIL]
      Cancellation in Main thread
      Expected: True
      Actual:   False
      Stack Trace:
        /_/src/libraries/System.ComponentModel.EventBasedAsync/tests/BackgroundWorkerTests.cs(126,0): at System.ComponentModel.EventBasedAsync.Tests.BackgroundWorkerTests.TestCancelAsync()
        /_/src/mono/System.Private.CoreLib/src/System/Reflection/RuntimeMethodInfo.cs(370,0): at System.Reflection.RuntimeMethodInfo.Invoke(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)
    System.ComponentModel.EventBasedAsync.Tests.BackgroundWorkerTests.TestFinalization [SKIP]
      Condition(s) not met: "IsPreciseGcSupported"
```

Re-running failed leg

**Review by karelz — APPROVED** [review]

File: src/libraries/System.Net.Security/tests/FunctionalTests/SslStreamStreamToStreamTest.cs (hunk 1 above)
Comment (ViktorHofer) [reply src/libraries/System.Net.Security/tests/FunctionalTests/SslStreamStreamToStreamTest.cs:70]: That's necessary for `SkipTestException` to work. The condition isn't statically defined but dynamically via throwing an Exception.

**Comment by ViktorHofer** [comment]
@karelz thanks for pointing out the unrelated test failure. What would help even more is if you could file an issue for it (in this case one doesn't exist yet) so that this information isn't lost in the nirvana of merged PRs.

**Comment by karelz** [comment]
@ViktorHofer I am hesitant to file one off test failures as they are super-noisy in Networking for example (we close them right away) -- IMO people searching for it will find the PR and if they see it 2nd time, they can create issue ... optimizing for process ;)

**Comment by ViktorHofer** [comment]
[The official guidance](https://github.com/dotnet/runtime/blob/main/docs/pr-guide.md#what-to-do-if-you-determine-the-failure-is-unrelated) has been to open issues for every unrelated test failure immediately so that area owners can look into the failures and triage them. In some cases such failures might just be noise (still our tests needs to handle that) but in other cases they might surface an underlying product issue.

File: src/libraries/System.Net.Security/tests/FunctionalTests/SslStreamStreamToStreamTest.cs (hunk 1 above)
Comment (wfurt) [reply src/libraries/System.Net.Security/tests/FunctionalTests/SslStreamStreamToStreamTest.cs:70]: BTW this is our own extension and that is also why Microsoft.DotNet.XUnitExtensions is needed @antonfirsov

**Pull request merged**